### ISSUE TITLE
New documented function get_tessdata seems to be missing

### ISSUE THREAD

**Issue by dingus9** [pymupdf/PyMuPDF#2500]
I'm not able to figure out where this function, or method is https://pymupdf.readthedocs.io/en/1.22.5/functions.html#get_tessdata

If it exists somewhere, I sure can't find it, searching code bases is coming up empty both in github against this project, grep on the installed python code and in python/pdb looking at the object provide by fitz.

If it exists somewhere odd, can you update the docs to specify how to access the function, otherwise maybe it doesn't exist?

**Comment by JorjMcKie** [comment]
You are right. Sorry, this function has been documented one patch version too early.
There is no big deal behind this feature. It will be the following code. Please feel free to already use it:
```python
def get_tessdata() -> str:
    """Detect Tesseract-OCR and return its language support folder.

    This function can be used to enable OCR via Tesseract even if the
    environment variable TESSDATA_PREFIX has not been set.
    If the value of TESSDATA_PREFIX is None, the function tries to locate
    Tesseract-OCR and return the required variable.

    Returns:
        Folder name of tessdata if Tesseract-OCR is available, otherwise False.
    """
    tessdata = os.getenv("TESSDATA_PREFIX")
    if tessdata != None:
        return tessdata 

    if sys.platform == "win32":
        tessdata = "C:\\Program Files\\Tesseract-OCR\\tessdata"
    else:
        tessdata = "/usr/share/tesseract-ocr/4.00/tessdata"

    if os.path.exists(tessdata):
        return tessdata

    """
    Try to locate the tesseract-ocr installation.
    """
    # Windows systems:
    if sys.platform == "win32":
        try:
            response = os.popen("where tesseract").read().strip()
        except:
            response = ""
        if not response:
            print("Tesseract-OCR is not installed")
            return False
        dirname = os.path.dirname(response)  # path of tesseract.exe
        tessdata = os.path.join(dirname, "tessdata")  # language support
        if os.path.exists(tessdata):  # all ok?
            return tessdata
        else:  # should not happen!
            print("unexpected: Tesseract-OCR has no 'tessdata' folder", file=sys.stderr)
            return False

    # Unix-like systems:
    try:
        response = os.popen("whereis tesseract-ocr").read().strip().split()
    except:
        response = ""
    if len(response) != 2:  # if not 2 tokens: no tesseract-ocr
        print("Tesseract-OCR is not installed")
        return False

    # determine tessdata via iteration over subfolders
    tessdata = None
    for sub_response in response.iterdir():
        for sub_sub in sub_response.iterdir():
            if str(sub_sub).endswith("tessdata"):
                tessdata = sub_sub
                break
    if tessdata != None:
        return tessdata
    else:
        print(
            "unexpected: tesseract-ocr has no 'tessdata' folder",
            file=sys.stderr,
        )
        return False
    return False
```

**Comment by JorjMcKie** [comment]
Resolved in version 1.23.2.
